be/src/olap/memtable_flush_executor.h

```diff
@@ -107,7 +107,7 @@ class MemTableFlushExecutor {
     void init(const std::vector<DataDir*>& data_dirs);
 
     Status create_flush_token(std::unique_ptr<FlushToken>* flush_token, RowsetTypePB rowset_type,
-                              bool is_high_priority);
+                              bool merge_on_write, bool is_high_priority);
```

Comment:
the new parameter should rename to `serial_mode`, MoW is a feature of unique key table, use `merge_on_write` here is confusing.